inc/db.function.php

```diff
@@ -644,6 +644,96 @@ function getTreeValueName($table, $ID, $wholename="", $level=0) {
  * Get the ancestors of an item in a tree dropdown
  *
  * @param $table     string   table name
+ * @param $items_id  array  The IDs of the items
+ *
+ * @return array of IDs of the ancestors
+**/
+function getAncestorsOfArray($table, $items_id) {
```

Comment:
Why would compatibility broken? If the existing function may take either an integer or an array; all existing code will still working as expected.
We probably need some tests on this.